kolibri/plugins/device/assets/src/views/FacilitiesPage/index.vue

```diff
@@ -290,7 +292,10 @@
         } else if (option === Options.REGISTER) {
           this.facilityForRegister = facility;
         } else if (option === Options.MANAGESYNC) {
-          this.$router.push({ path: '/manage' });
+          this.$router.push({
```

Comment:
This is good - you can also attach an event handler to the `FacilityNameAndSyncStatus` to handle the `manageSync` event to navigate to this page as well, so that both means for getting to this page are handled.